### PR TITLE
Ekco 0 5 0

### PR DESCRIPTION
**Pull request by areed**

### CODE REVIEW

File: addons/ekco/0.5.0/install.sh [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+function ekco() {
+    local src="$DIR/addons/ekco/$EKCO_VERSION"
+    local dst="$DIR/kustomize/ekco"
+
+    cp "$src/kustomization.yaml" "$dst/kustomization.yaml"
+    cp "$src/namespace.yaml" "$dst/namespace.yaml"
+    cp "$src/rbac.yaml" "$dst/rbac.yaml"
+    cp "$src/rolebinding.yaml" "$dst/rolebinding.yaml"
+    cp "$src/deployment.yaml" "$dst/deployment.yaml"
+    cp "$src/rotate-certs-rbac.yaml" "$dst/rotate-certs-rbac.yaml"
```
Comment (areed) [inline addons/ekco/0.5.0/install.sh:39]: New since 0.4.2

File: addons/ekco/0.5.0/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace: kurl
+
+resources:
+- namespace.yaml
+- rbac.yaml
+- deployment.yaml
+- rotate-certs-rbac.yaml
```
Comment (areed) [inline addons/ekco/0.5.0/kustomization.yaml:7]: New since 0.4.2

File: addons/ekco/0.5.0/rotate-certs-rbac.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (areed) [inline addons/ekco/0.5.0/rotate-certs-rbac.yaml:1]: This whole file is new since 0.4.2

File: addons/ekco/0.5.0/tmpl-configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    maintain_rook_storage_nodes: $EKCO_SHOULD_MAINTAIN_ROOK_STORAGE_NODES
+    ceph_block_pool: replicapool
+    ceph_filesystem: rook-shared-fs
+    ceph_object_store: rook-ceph-store
+    min_ceph_pool_replication: 1
+    max_ceph_pool_replication: 3
+    certificates_dir: /etc/kubernetes/pki
+    reconcile_interval: 1m
+    rook_version: $ROOK_VERSION
+    rotate_certs_image: replicated/ekco:v$EKCO_VERSION
```
Comment (areed) [inline addons/ekco/0.5.0/tmpl-configmap.yaml:21]: Here to end of file is new since 0.4.2

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**